hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -695,6 +695,13 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+sw_test_timeout_ns=15000000"]
     }
+    {
+      name: chip_sw_csrng_kat_test
+      uvm_test_seq: chip_sw_base_vseq
+      sw_images: ["sw/device/tests/csrng_kat_test:1"]
+      en_run_modes: ["sw_test_mode_test_rom"]
+      run_opts: ["+sw_test_timeout_ns=18000000"]
```

Comment:
I knew entropy could be slow, but this is a large timeout... Is it realistic?

Reply:
I will take a look. I suspect it will need to be updated.